pkg/cluster/internal/kubeadm/config.go

```diff
@@ -125,29 +148,29 @@ kubeletConfiguration:
       nodefs.available: "0%"
       nodefs.inodesFree: "0%"
       imagefs.available: "0%"
-controllerManagerExtraArgs:
-  enable-hostpath-provisioner: "true"
```

Comment:
related/intentional?

Reply:
It moved down to merge with the feature gates. A previous commit toggling DynamicProvisionScheduling failed to combine these.

Reply:
Ah, missed it. Surprised strict parsing didn't catch the duplicate key

Reply:
We have strict parsing on our own config but not on any upstream k8s yet as far as I know.